### DISCUSSION TITLE
Panel table view

### DISCUSSION THREAD

**Discussion by worksofliam** [microsoft/vscode-discussions#38] [Extension Development QnA] [answered]
So now the problems view supports showing results in a table as well as a treeview format, does that mean we can also utilize panels and allow our extensions to contribute a table view?

**Comment by usernamehw** [comment, marked as answer]
Nope, they are not yet exposed to extensions https://github.com/microsoft/vscode/issues/118115

**Reply by fselcukcan** [reply]
It would be good to filter problems by source, e.g. eslint, ts (typescript server) etc, or being able to write an extension to do that.
